### DISCUSSION TITLE
How to make an "Override" mode

### DISCUSSION THREAD

**Discussion by Krutors** [Rem0o/FanControl.Releases#2716] [General]
Hi
Is this already possible to make a custom curve or parameter that override every other one when a specific condition is met ? 

I'm reporting my liquid temp to FC, but can't semm to find a way to make my fans and pump go full bore if said liquid go above a certain temp, while having other custom curves active at the same time.

The closest thing I can think about wich could fill that role is a "Trigger" fuction, but with curves as inputs, on top of sensors.

**Comment by Rem0o** [comment]
It's called the "Mix" fan curve.   Play with it, it's exactly what you ask for.

**Reply by Krutors** [reply]
Hi Rémi,

Thanks for the fast response.
While I was trying to play with the "Mix" Curve, I may have encountered a bug : it seems like if the user create an Offset sensor, t's value is not taken into account inside the "Mix". Is it because the two entries of the mix have the same sensor as a base ?

![FC](https://github.com/user-attachments/assets/ecc637b0-6d79-4441-ba74-14d3cdb6f836)

**Reply by Rem0o** [reply]
No it's because both curves need to be in rpm mode. You can't mix % and rpm.
